cli/src/diff_util.rs

```diff
@@ -1244,21 +1254,28 @@ pub fn show_diff_stat(
     display_width: usize,
 ) -> Result<(), DiffRenderError> {
     let mut stats: Vec<DiffStat> = vec![];
+    let mut unresolved_renames = HashSet::<String>::new();
```

Comment:
This is a stylistic nit, this can just be `HashSet::new` as Rust will infer it on line 1276.

Reply:
done